CGMBLEKit/TransmitterManager.swift

```diff
@@ -335,6 +335,11 @@ public class TransmitterManager: TransmitterDelegate {
             ))
         }
 
+        // Filter out future-dated events
+        events = events.filter { event in
```

Comment:
This is not fixing the underlying problem (incorrect parsing of message data earlier in the process).  Please add a comment that this is a stopgap fix for the issue (https://github.com/LoopKit/Loop/issues/2087) and add an error level log here that includes a string description of the `glucose` object.